packages/common/src/sites/searchCategoriesToCollections.ts

```diff
@@ -0,0 +1,90 @@
+import { IHubCollection } from "../search/types/IHubCatalog";
+import { WellKnownCollection } from "../search/wellKnownCatalog";
+import { SearchCategories } from "./_internal/types";
+import { defaultSiteCollectionKeys } from "./defaultSiteCollectionKeys";
+
+/**
+ * Convert searchCategories to collections. If no searchCategories are provided,
+ * the default collections will be returned.
+ *
+ * @param searchCategories searchCategories from site model (data.values.searchCategories)
+ * @returns IHubCollections that respect the searchCategories configuration
+ */
+export function searchCategoriesToCollections(
```

Comment:
I needed to expose this function, so the content library could use it. There's nothing new in the function, it's just copied out out `applyDefaultCollectionMigration()`. I didn't add tests because this function is covered by the existing tests in `applyDefaultCollectionMigration()`.